### PR TITLE
Mention organizers

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Need approvals from all of you since this pertains the "us" (lowercase so it doesn't get confused with the country 😄).

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -65,12 +65,12 @@ languages:
...
     image: img/languages/otpcl.svg
 
-#organizer:
-#  name: Fiqus
-#  image: sponsors_2020/fiqus.png
-#  logo: sponsors_2020/fiqus.png
-#  url: https://fiqus.coop/en/
-#  background_color: "#FFFFFF"
+organizer:
+  name: Some kind folks of the BEAM community
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:69]: I also though about "Some nice like-minded folks...": suggestions accepted.

File: _config.yml [hunk 2]
```diff
@@ -65,12 +65,12 @@ languages:
...
+#  image:
+#  logo:
+  url: "/#organizers"
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:72]: Links to footer.

File: _config.yml [hunk 3]
```diff
@@ -111,11 +111,11 @@ languages:
 #    background_color: "#510512"
 
 silver_sponsors:
-  - name: NineFX
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:114]: Unrelated, but somehow not pushed previously...

File: _includes/sponsor_logos.html [hunk 4]
```diff
@@ -53,28 +53,3 @@
         {% endfor %}
     </div>
 </div>
-
-<div class="container brands">
-    <div class="d-flex justify-content-center align-items-center sponsors">
```
Comment (paulo-ferraz-oliveira) [inline _includes/sponsor_logos.html:58]: This first div is unrelated, but removing the second one already meant having to fix this block, so yeah... (it'll surely cause a merge conflict with my other to-be-approved pull request).

File: _includes/sponsors.html [hunk 5]
```diff
@@ -90,13 +90,17 @@ <h2 class="display-5 col-12 text-center ">Media</h2>
     </div>
     {% endif %}
 
-    <!-- Organizer Sponsor -->
```
Comment (paulo-ferraz-oliveira) [inline _includes/sponsors.html:93]: Removed mentions to sponsor.

File: _includes/sponsors.html [hunk 6]
```diff
@@ -90,13 +90,17 @@ <h2 class="display-5 col-12 text-center ">Media</h2>
...
+    <!-- Organizer -->
     {% if site.organizer.name %}
     <div class="row mt-3 sponsors-container">
         <h2 class="display-5 col-12 text-center">Organizer</h2>
-        <a class="col-md-3 col-sm-12" href="{{ site.organizer.url }}">
-            <div class="rounded mx-auto text-center sponsor sponsor-organizer " style="background-color: {{ site.organizer.background_color }}; ">
+        <a class="col-md-3 col-sm-12 organizer" href="{{ site.organizer.url }}">
+            <div class="rounded mx-auto text-center" style="background-color: {{ site.organizer.background_color }}; ">
+                {% if site.organizer.image %}
                 <img class=" p-2 " src="/img/{{ site.organizer.image }} " alt="{{ site.organizer.name }} ">
```
Comment (paulo-ferraz-oliveira) [inline _includes/sponsors.html:100]: One day we might even have an image, who knows.

File: css/style.css [hunk 7]
```diff
@@ -906,3 +910,22 @@ footer a:hover {
...
+    border-radius: 50% !important;
+    height: 32px;
+    opacity: 80%;
+}
+.brought-to-you-by-imgs a,a:hover {
+    color: white;
+    vertical-align: bottom;
+}
+
+.brought-to-you-by-imgs a:not(:last-child) {
```
Comment (paulo-ferraz-oliveira) [inline css/style.css:929]: Mad CSS skills 😄

File: _config.yml (hunk 2 above)
Comment (filipevarjao) [reply _config.yml:72]: github is fine ~> https://github.com/filipevarjao

Comment (filipevarjao) [reply _config.yml:72]: or just the names, anything works for me

**Review by starbelly — APPROVED** [review]

File: _config.yml (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply _config.yml:72]: It's what you'll find in the footer. 😄

**Review by pablocostass — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Squashing, merging, deploying...

**Pull request merged**